source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak.md

```diff
@@ -191,70 +252,84 @@ with ports used by another application or if you use a load balancer or
 proxy server.
 </div>
 
-It is also recommended that you increase the size of Riak's `pb_backlog`
-to be greater than the size of `request_pool` specified in the Riak CS
-`app.config` file. At minimum, `pb_backlog` should be set to `64`. The
-default value is `5`. To do so, uncomment the appropriate line and
-replace `{pb_backlog, 5}` with this:
-
-```appconfig
-{riak_api, [
-            %% Other configs
-            {pb_backlog, 64},
-            %% Other configs
-           ]}
-```
+It is also recommended that users insure that the size of Riak's
+`protobuf.backlog` (or in the `advanced.config`/`app.config` files, the
+`pb_backlog`) is equal to or greater than the size of the
+`pool.request.size`, specified in the Riak CS `riak-cs.conf` (or
+the `request_pool` size in the `advanced.config`/`app.config` files).
 
-If the `request_pool` value in Riak CS is changed, the `pb_backlog`
+If the `pool.request.size` value in Riak CS is changed, the `protobuf.backlog`
 value in Riak should be updated as well.
 
 ## Other Riak Settings
 
-The `app.config` file includes other settings, such as turning on the
-creation of log files and specifying where to store them. These
+The `riak.conf` and `advanced.config` files includes other settings, such as
+turning on the creation of log files and specifying where to store them. These
 settings have default values that should work in most cases. For more
-information, we recommend reading our [configuration files](:w
-) doc
+information, we recommend reading our [configuration files](:w) documentation.
 
 ## Specifying the Riak IP Address
 
-By setting the Riak IP address you ensure that your Riak nodes have
-unique IP addresses, whether you're working with a single node or adding
-additional nodes to the system. The Riak IP address setting resides in
-the Riak `vm.args` configuration file, which is located in the same
-`/etc/riak` folder as `app.config` (or in `/opt/riak/etc` on some
-operating systems).
+By setting the Riak IP address you ensure that your Riak nodes have unique IP
+addresses, whether you're working with a single node or adding additional nodes
+to the system. The Riak IP address setting resides in the Riak `riak.conf` or
+the old-style `vm.args` configuration file, which is located in the same
```

Comment:
Here we're using old-style to describe an entirely different syntax. Not sure that's a good idea.